foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,158 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man, src
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+import inspect
+from Foam.OpenFOAM import fileName, word, IOobject
+from collections import defaultdict
+from Foam.finiteVolume import volScalarField
+from Foam.src.OpenFOAM import *
+
+class FoamWrapper(object):
```

Comment:
Do we already have some version of  the abstract class ABCModelingEngineWrapper in cuds github ?